### PR TITLE
Refactor zuora properties - remove old properties

### PR DESCRIPTION
**Pull request by jonflynng**
As mentioned in the [previous PR](https://github.com/guardian/manage-frontend/pull/947). This PR removes the old properties.

 The best way to review this PR would be to do a diff of this branch and the main branch before my last PR, and assert that the property names have changed accordingly.

I have also added a new Cypress test for the update contribution amount journey:

Screenshot 2023-01-06 at 14 48 14

### CODE REVIEW

File: client/components/mma/billing/Billing.tsx [hunk 1]
```diff
@@ -151,13 +151,10 @@ const BillingRenderer = ([mdaResponse, invoiceResponse]: [
...
 										}));
-									const resultsPerPage = (
-										paidPlan.billingPeriod ||
-										paidPlan.interval ||
-										''
-									)?.includes('year')
-										? productInvoiceData.length
-										: 6;
+									const resultsPerPage =
+										paidPlan.billingPeriod?.includes('year')
```
Comment (jonflynng) [inline client/components/mma/billing/Billing.tsx:155]: not sure this looks right, ill investigate

File: client/components/mma/billing/Billing.tsx [hunk 2]
```diff
@@ -151,13 +151,10 @@ const BillingRenderer = ([mdaResponse, invoiceResponse]: [
 											productUrlPart:
 												specificProductType.urlPart,
 										}));
-									const resultsPerPage = (
-										paidPlan.billingPeriod ||
-										paidPlan.interval ||
-										''
-									)?.includes('year')
```
Comment (jonflynng) [inline client/components/mma/billing/Billing.tsx:158]: not sure why this is optional. I've checked the main branch before my refactors and it seems it always has been, but it is required on the type interface.

**Comment by jonflynng** [comment]
[Rename object properties to be in-line with Zuora's naming](https://trello.com/c/ZfJMciLT/3737-rename-object-properties-to-be-in-line-with-zuoras-naming)

File: client/components/mma/billing/Billing.tsx (hunk 2 above)
Comment (andrewHEguardian) [reply client/components/mma/billing/Billing.tsx:158]: No idea why either, seems fine to leave as it was for now

**Pull request merged**